src/Compilers/Core/Portable/Binding/BindingDiagnosticBag.cs

```diff
@@ -207,23 +207,29 @@ internal void AddDependencies(CompoundUseSiteInfo<TAssemblySymbol> useSiteInfo)
             }
         }
 
+        internal bool Add(Location location, CompoundUseSiteInfo<TAssemblySymbol> useSiteInfo)
```

Comment:
>internal bool Add(Location location, CompoundUseSiteInfo<TAssemblySymbol> useSiteInfo) [](http://example.com/codeflow?start=8&length=86)

As a general feedback. Reordering of members makes it significantly harder to review the change. Please refrain from doing so. Order of members can be changed in a pure refactoring PR before after the change that makes functional changes.